### PR TITLE
Creates Carbon Costs doc

### PR DESCRIPTION
**Pull request by bstuder99**

### CODE REVIEW

File: using-kubecost/carbon-costs.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Carbon Costs
+
+Carbon Costs is a cost metric added to both [Allocation](/using-kubecost/navigating-the-kubecost-ui/cost-allocation/README.md) and [Assets](/using-kubecost/navigating-the-kubecost-ui/assets.md) dashboards. Carbon Costs are measured in KG CO2e, which is defined by the EPA as:
+
+> Carbon dioxide equivalent or CO2e means the number of metric tons of CO2 emissions with the same global warming potential as one metric ton of another greenhouse gas, and is calculated using Equation A-1 in 40 CFR Part 98.
+
+For more information on how carbon costs are calculated, visit the [Cloud Carbon Footprint](https://www.cloudcarbonfootprint.org/) website.
```
Comment (dwbrown2) [inline using-kubecost/carbon-costs.md:7]: @kaelanspatel @bstuder99 what about a 1-2 line summary about how these costs are calculated/estimated?

File: using-kubecost/carbon-costs.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Carbon Costs
+
+Carbon Costs is a cost metric added to both [Allocation](/using-kubecost/navigating-the-kubecost-ui/cost-allocation/README.md) and [Assets](/using-kubecost/navigating-the-kubecost-ui/assets.md) dashboards. Carbon Costs are measured in KG CO2e, which is defined by the EPA as:
```
Comment (dwbrown2) [inline using-kubecost/carbon-costs.md:3]: How about 1-2 lines on why these metrics are important? Some copy ideas here: https://www.opencost.io/blog/carbon-costs

**Review by dwbrown2 — COMMENTED** [review]
Looks like a good start! @kwombach12 @kaelanspatel @bstuder99 are we planning to link to this from within the product anywhere.

**Comment by kwombach12** [comment]
> Looks like a good start! @kwombach12 @kaelanspatel @bstuder99 are we planning to link to this from within the product anywhere.

@dwbrown2 we added this description directly in a tooltip

File: using-kubecost/carbon-costs.md (hunk 1 above)
Comment (kaelanspatel) [reply using-kubecost/carbon-costs.md:7]:
```suggestion
Kubecost uses carbon coefficient data generated by the [Cloud Carbon Footprint](https://www.cloudcarbonfootprint.org/) to associate estimated carbon costs to Disk, Node, and Network Assets based on their runtime. This carbon cost is then distributed to Allocations. For more information on how these coefficients are calculated, visit the Cloud Carbon Footprint's [methodology page](https://www.cloudcarbonfootprint.org/docs/methodology).
```
Definitely. The coefficients themselves come from the CCF.

**Comment by bstuder99** [comment]
@kwombach12 It might be safe to confirm with Neal, but as far as I know, there's no need to link to this article from the product UI. Hovering the cursor over the Carbon Costs column will provide a short description of the metric and that's it. If you'd like it added, FE can do that. The slug will look like: https://docs.kubecost.com/using-kubecost/carbon-costs

**Comment by bstuder99** [comment]
@kaelanspatel Thank you for the updated description, that's what I was looking for. Let me know if this can be approved.

**Pull request merged**